### ISSUE TITLE
Model:Predictions.

### ISSUE THREAD

**Issue by Tikam02** [leeyeehoo/CSRNet-pytorch#6]
How to predict estimated count from the trained model in Val.ipynb.
Where is the output of the predicted estimated count?
@leeyeehoo

**Comment by leeyeehoo** [comment]
Well I have to refine it later, it's just in the source code... Sorry for that

**Comment by Tikam02** [comment]
![screenshot from 2018-07-21 21-02-55](https://user-images.githubusercontent.com/10548085/43037265-837f0c5a-8d29-11e8-8b60-5deacef0732c.png)


Well, I created  density map for the different image from the model that I have saved. But couldn't able to find how to predict numbers.

**Comment by leeyeehoo** [comment]
You should use make_dataset.ipynb to generate the density map. And all the density map will be saved into .h5 file independently. You can just read the density map from the .h5 and use np.sum() func to calculate the numbers.

**Comment by liuleiBUAA** [comment]
why my mae so high
0 1039.234375
1 3270.0751953125
2 4630.0849609375
3 7451.38427734375
4 13182.578125
5 14520.990844726562
6 20020.381469726562
7 34471.97229003906
8 36625.15148925781
9 41896.54553222656
10 51193.14318847656
11 51800.69665527344
12 52128.655670166016
13 52754.70217895508
14 55443.40872192383
15 56253.32388305664
16 60179.312408447266
17 60813.217376708984
18 63668.902923583984
19 66381.42611694336
20 69399.40219116211
21 71583.58383178711
22 78590.99594116211
23 82238.25765991211
24 83930.38412475586
25 91782.86410522461

**Comment by liuleiBUAA** [comment]
@leeyeehoo when I train the model with train.py the loss is nan

Epoch: [0][1170/1200] Time 0.709 (0.417) Data 0.020 (0.017) Loss nan (nan)

here is the loss in terminal, I comment the seen=model.seen, otherwise it will give the error of no attribute of model.seen
